pkg/ingester/metrics.go

```diff
@@ -203,6 +207,17 @@ func newIngesterMetrics(r prometheus.Registerer, createMetricsConflictingWithTSD
 		}, []string{"user"})
 	}
 
+	if !createMetricsConflictingWithTSDB {
+		m.numUsersWithBackfillTSDBs = promauto.With(r).NewGauge(prometheus.GaugeOpts{
+			Name: "cortex_ingester_tsdb_users_with_backfill_tsdb",
+			Help: "Total number of users with backfill TSDBs",
+		})
+		m.numBackfillTSDBsPerUser = promauto.With(r).NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cortex_ingester_tsdb_backfill_tsdb_per_user",
```

Comment:
I would suggest `cortex_ingester_backfill_tsdb_opened`. The dimension "user" shouldn't be part of the metric name, so I would not mention "per_user" in the metric name.

Reply:
I will keep it `cortex_ingester_backfill_tsdb_open` as this is a gauge (current open TSDBs)